app/_landing_pages/gateway/network.yaml

```
@@ -0,0 +1,33 @@
+metadata:
+  title: "{{site.base_gateway}} network, ports, and firewall"
```

```suggestion
  title: "{{site.base_gateway}} networking"
```
Networking encompasses all of the other pieces (ports, firewall info, etc), so we can just name the landing page that.